### ISSUE TITLE
Attribute being replaced

### ISSUE THREAD

**Issue by ogamacheInk** [verbb/formie#1207]
### Describe the bug

When making a form, with acustom rendering form, a field with a custom attibute like `type="password"` wont have a `name` attribute.

### Steps to reproduce

1. Create form with an attribute (type=password)
2. Render the form on a page using the custom rendering form for login
3. Get the error message (Request missing required body param)

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

Craft Pro 4.3.4

### Plugin version

2.0.19

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.19",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.20](https://github.com/verbb/formie/releases/tag/2.0.20)
